core/store/migrate/migrations/0158_update_job_proposal_spec_job_proposal_statuses.sql

```diff
@@ -0,0 +1,144 @@
+-- +goose Up
+-- +goose StatementBegin
+-- JobProposals Table
+-- We must remove the old contraint to add an enum value to support Postgres v11
+ALTER TABLE job_proposals DROP CONSTRAINT chk_job_proposals_status_fsm;
```

Comment:
Since you are only adding you can do this now since PG 9.1

`ALTER TYPE enum_type ADD VALUE 'new_value';`

Reply:
So used to the rename recreate reassign drop flow of updating a PG enum

Reply:
Unfortunately, this won't allow me to update the null constraint check in the same migration as I get `Unsafe use of new value of enum type` error for trying to use the new constraint in the same transaction.